dwave/optimization/generators.py

```diff
@@ -407,6 +409,243 @@ class as the decision variable being optimized, with permutations of its
     return model
 
 
+def capacitated_vehicle_routing_with_time_windows(demand: numpy.typing.ArrayLike,
+                                                  number_of_vehicles: int,
+                                                  vehicle_capacity: float,
+                                                  time_distances: typing.Optional[numpy.typing.ArrayLike] = None,
+                                                  time_window_open: typing.Optional[numpy.typing.ArrayLike] = None,
+                                                  time_window_close: typing.Optional[numpy.typing.ArrayLike] = None,
+                                                  service_time: typing.Optional[numpy.typing.ArrayLike] = None,
```

Comment:
If these are allowed to be `None`, the docs should explain what they get defaulted to.